tests/CoreFX/CoreFX.issues.json

```diff
@@ -1128,6 +1128,26 @@
             "namespaces": null,
             "classes": null,
             "methods": [
+                {
+                    "name": "System.Tests.AppDomainTests.MonitoringIsEnabled",
```

Comment:
@jkotas, has the suppression mechanism changed?  These tests are still showing up at https://mc.dot.net/#/user/coreclr-ci/pr~2Fdotnet~2Fcoreclr~2Frefs~2Fpull~2F24610~2Fmerge/test~2Ffunctional~2Fcli~2F/20190520.764/workItem/System.Runtime.Extensions.Tests as failing, even though I think I've suppressed them here correctly.  Do you see anything I've done incorrectly?

Reply:
Yes, the exclusion file was changed recently. I have updated the docs: https://github.com/dotnet/coreclr/commit/d498abd2ec0937bbc5005d2d8d51add89c58a946 . I will submit PR to cleanup files that are no longer used.

Reply:
Aha, thanks, I'll fix the PR.

Reply:
https://github.com/dotnet/coreclr/pull/24684